<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminstrator</class>
 <widget class="QDialog" name="adminstrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1021</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>ADMINSTRATOR</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Adminstrator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add user</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removed_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove user</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCourse">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>178</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>add course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeCourse">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>remove course</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="removed">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addCourse1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="removeCourse1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
